src/splitseq_mkref.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+#USAGE: ./splitseq_mkref.sh "genome names" "fasta files" "gtf files"
+
+genome_names=$1  #space-delimited, e.g. "hg38 mm10"
+fastas=$2        #space-delimited, e.g. "hg38.fa mm10.fa"
+gtfs=$3          #space-delimited, e.g. "hg38.gtf mm10.gtf"
+
+splitseq_venv="/Genomics/grid/users/rleach/local/splitseq"
+splitseq_refs="${splitseq_venv}/splitseq_refs"
+refdir="${splitseq_refs}/"`echo "${genome_names}" | perl -pne 's/ /_/g;'`
+
+cd "${splitseq_venv}"
```

Comment:
Activate environments instead of `cd` to them. Also, generally, I'd prefer the user to setup the environment and have the script assume things are in the path. That way, the user can use modules, conda env, or python venv as they see fit.

Reply:
Will do.  I'm still a noob with this environment stuff.  Will do.

Reply:
So there's no conda package for split-seq.  If I don't do a cd, what's the best way to supply the split-seq script to their python call?